grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -65,6 +65,8 @@ class StockMovementService {
 
     boolean transactional = true
 
+    def currentDate
```

Comment:
I don't think you can use it like this, it's not thread safe (the same variable will be used by all of the users)